src/pages/Dashboard/Dashboard.tsx

```diff
@@ -62,20 +46,16 @@ const ScrollView = (props: FlexProps) => (
 
 const VirtualizedSwipableViews = virtualize(SwipeableViews)
 
-// Leverage the fact that enums don't exist in native JS and compile to 0, 1, and 2 when accessed here
+// Leverage the fact that enums don't exist in native JS and compile to 0 and 1 when accessed here
 // so we can have a declarative way to refer to the tab indexes instead of magic numbers
 enum MobileTab {
-  Overview,
-  Earn,
-  Activity,
+  MyCrypto,
+  Watchlist,
 }
 
 export const Dashboard = memo(() => {
   const translate = useTranslate()
   const [slideIndex, setSlideIndex] = useState(0)
-  const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`, { ssr: false })
-  const appIsMobile = isMobile || !isLargerThanMd
```

Comment:
No need for this insanity, this is actually super confusing: `!isLargerThanMd` is a superset of `isMobile` for the mobile app case, so we only needed the latter for responsive things. 
Went with `<Display.Mobile />` instead, which is effectively a wrapper over `!isLargerThanMd` and which is IMO more ergonomic.